Add EuroPython, Python Web Conf and make location/venue uniform

vinayak-mehta opened the pull request:
Added EuroPython and Python Web Conf.

In addition to that, I've updated the location and venue for the following conferences for uniformity. Any feedback on the naming is appreciated, I will make the necessary changes.

- Remote Python Pizza
- Scipy 2020
- PyOhio
- PyConline AU
- PyGotham TV
- PyCon India
- PyCon Sweden

Also, DragonPy and ~PyCon ES~ have been postponed to next year. What should 2020.csv reflect in that case? Should these conferences be moved to 2021.csv?

Edit: PyCon ES is not postponed, I initially saw that announcement on the link in 2020.csv: https://2020.es.pycon.org. But it is happening as an online event in October: https://es.python.org/pycones-2020-call-for-proposals-is-open.html

vinayak-mehta commented:
Are conferences based on Python projects like DjangoCon, JupyterCon and Airflow Summit within the scope of this repo?

Mariatta commented:
We have added DjangoCon in the past, I think it belongs here. So as JupyterCon. I don't know about Airflow summit though.

vinayak-mehta commented:
Thanks for the review. I've reverted the "Earth" change, added the initial planned locations for EuroPython, Pyconline AU and PyCon India; and changed the venue for Scipy to Online. I've left the locations for Python Web Conf and PyCon Sweden as blank since I'm not sure what to fill there.

Mariatta commented:
Python Web Conf is based in Fishers, Indiana, USA and PyCon Sweden is based in Stockholm, Sweden

vinayak-mehta commented:
> We have added DjangoCon in the past, I think it belongs here. So as JupyterCon.

Should I add DjangoCon EU, FlaskCon and JupyterCon in the same PR?

> I don't know about Airflow summit though.

IMO if we can add the above conferences, then we should include Airflow summit too, as this Python project caters to a subset of the Python community (data engineering) similar to how Django, Flask and Jupyter cater to the web development and data science subsets.

vinayak-mehta commented:
> Python Web Conf is based in Fishers, Indiana, USA and PyCon Sweden is based in Stockholm, Sweden

Thanks! I'll add these locations.

vinayak-mehta commented:
Thanks for asking for the revert on "Earth", I think it makes sense to preserve the initial locations for all conferences happening online.

Mariatta commented:
Thanks. I'm not familiar with Airflow or Airflow summit, so perhaps folks can chime in on it.

vinayak-mehta commented:
Got it, thanks. I'll raise another PR for DjangoCon, FlaskCon and JupyterCon then.
